docs/revsets.md

```diff
@@ -70,6 +70,38 @@ only symbols.
 You can use parentheses to control evaluation order, such as `(x & y) | z` or
 `x & (y | z)`.
 
+### The `all:` prefix
+
+An `all:` prefix before a revset expression does not change its meansing.
+Strictly speaking, it is not part of the revset language. The notation is
+similar to the prefixes allowed before [string patterms](#string-patterns).
+
+Certain commands (such as `jj rebase`) can take multiple revset arguments, and
+each of these may resolve to one-or-many revisions. By default, `jj` will not
+allow revsets that resolve to more than one revision &mdash; a so-called "large
+revset" &mdash; and will ask you to confirm that you want to proceed by
+prefixing it with the `all:` modifier.
```

Comment:
This paragraph should probably go first, before the proceeding two? And the first two paragraphs can just be collapsed into one.

Reply:
Done, thanks!